### ISSUE TITLE
There is no column with name 'parent_id' on table 'role'.

### ISSUE THREAD

**Issue by dennis-fedco** [bjyoungblood/BjyAuthorize#206]
When running:

```
$ vendor/bin/doctrine-module orm:validate-schema
```

I get this error:

```
[Mapping]  OK - The mapping files are correct.

[Doctrine\DBAL\Schema\SchemaException]
There is no column with name 'parent_id' on table 'role'.
```

That is incorrect as there is a column named `parent_id` in my table `role`.

In my `composer.json` I am using:

```
"bjyoungblood/bjy-authorize": "~1.4"
```

I was getting the error, so I updated my User and Role default entities to the latest from branch `master` and still same error.  I use doctrine-module to generate my entities.

What could be wrong?  I looked at similar issues, but not sure I got my answer.

**Comment by Jngai** [comment]
I am getting that error even though I set it as a root user. Unfortunately that link is broken now.

**Comment by dennis-fedco** [comment]
Try this link https://github.com/doctrine/dbal/issues/1990
In my case I thought permissions were okay when they were not.  Make sure that your user that is accessing the database can access it.  i.e. log into your database as the same user as specified in your Doctrine configuration file, and then do a `SELECT * from Table` command.  If that works out fine than you may have a different (new) issue.

**Comment by Jngai** [comment]
I had this problem within a laravel app setting. I did everything, uninstall/reinstall mysql, created a new user, give grants to that database/database table to this user, kept on dropping/recreating db and dropping migrations table. The solution for me is to rewrite my laravel migrations files into sql statements. One of my migration table is like this

```
Schema::table('categories', function ($table) {
   $table->renameColumn('section', 'category');
});
```

I rewrote the add column laravel syntax into sql 

```
\DB::statement('ALTER TABLE categories CHANGE section category VARCHAR(255)');
```
